games.json

```diff
@@ -1501,18 +1501,24 @@
     "name": "World War 3",
     "logo": "",
     "native": false,
-    "status": "Broken",
-    "reference": "https://github.com/ValveSoftware/Proton/issues/1811#issuecomment-981078015",
+    "status": "Running",
+    "reference": "https://www.protondb.com/app/674020#I0Azq2ZzdO",
     "anticheats": [
-      "Mail.ru Anti-Cheat"
+      "Easy Anti-Cheat"
     ],
     "notes": [],
-    "updates": [],
+    "updates": [
+	    { 
+		"name": "Switched to Easy Anti-Cheat" 
```

Comment:
just wanted to flag these lines are missing the `,` at the end, it's still invalid JSON

Reply:
@Curve @Drsheppard01